### ISSUE TITLE
Custom-search causing other filters not to be applied depending on order

### ISSUE THREAD

**Issue by Verdoso** [buefy/buefy#3314]
Hi,
When I define a custom search function for a table column, then something weird happens: If I filter first by that column, then if I add another filtering criteria in a different column it is not applied. On the other hand, if I start filtering by this second column and then add the criteria to the column that has the custom-search attribute, both filters are applied as expected.
In the first case, no errors appear in the console. The filter is simply not applied and no changes happen to the rows.

It's reproducible 100% with current version.
I created a codepen based on the current buefy documentation for the table element so it is easy to test:

[Issue reproduced](https://codepen.io/Verdoso/pen/VwPoMZV?editors=1010)

Set 'J' as first name filter, then set 'S' as second name filter: Everything works as expected.
Set 's' as second name filter, then whatever you set as first name filter, nothing is filtered out.

**Buefy** version: v0.9.7
**Vuejs** version: v2.6.12
**OS/Browser**: Tested with Firefox/Chrome, Windows 10

### Steps to reproduce

The only thing you need to do is to add a custom-search function to a searchable column.

### Expected behavior

I would expect both filters to be applied, no matter what order I introduce them.

**Comment by Sim175** [comment]
I'm having this bug as well.
I'm not comfortable enough with the toolchain used to fix it myself, but I believe line 1154 here is wrong: https://github.com/buefy/buefy/blob/dev/src/components/table/Table.vue#L1154

It should be similar to line 1159: `if (!column.customSearch(row, input)) { return false }`, to allow subsequent filters to be applied.

**Comment by Tom2002** [comment]
Hey, i also noticed this bug. In my case i use custom-search for all my columns. In this case only the first filter you write is applied. All subsequent filters are ignored because my custom-search function is not called at all for the other columns.

**Comment by Verdoso** [comment]
Hello there,
I downloaded the dev bracn to have a go at the issue myself and I was unable to reproduce it... I then checked and the issue is no more in version 0.9.10. Not sure when it was fixed, but it seems it is no longer an issue, I'm going to close it.

TLDR: **Fixed in 0.9.10**
